MetaMask/metamask-extension
# test: Add dapp interaction of wallet_revokePermissions for eth_accounts

## **Description**

E2E test for `wallet_revokePermissions` for `eth_accounts` via dapp interaction.



## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1706

## **Manual testing steps**

1. Pull down this branch
2. Run `yarn && yarn build:test && yarn test:e2e:single test/e2e/tests/revoke-permissions.spec.js --browser=chrome`
3. Should pass
4. Optional run full e2e test`yarn test:e2e:chrome` should pass
## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
